### ISSUE TITLE
Missing info icon is slow to load for large rosters

### ISSUE THREAD

**Issue by RyanHansz** [ctoec/data-collection#457]
**Update - Nov 10** All missing info icons show up now, but take a significant time to load, making it seem like the records don't have missing info. 

The icon has also gotten super small on the roster, should be 14x14px.

**Comment by RyanHansz** [comment]
Possibly related to these errors:
![Uploading Screen Shot 2020-10-06 at 1.57.28 PM.png…]()

**Comment by RyanHansz** [comment]
Look again once the pagination is implemented to see if this is still an issue

**Comment by jlhogan** [comment]
The way pagination ended up being implemented I don't think will really help -- I think the best path forward is to inline the SVG content in their components so that the client isn't making fetch requests to get the files every time. 
I tried doing something with svg `use` a la https://css-tricks.com/svg-use-external-source/, but it still was re-fetching the xml file for every single rendered svg icon. I had better luck with rewriting the react components themselves to just include the svg content. not sure if there are downsides to that. can dig up that branch and put it up as something to discuss / react to

**Comment by jlhogan** [comment]
tldr of above: make exclamation point in circle an svg in the component lib (not just inherited from uswds) and import it as `ReactComponent` in the client to drastically speed up rendering of the svg by leveraging caching & avoiding an individual fetch for each separate icon
